common-npm-packages/azure-arm-rest-v2/webClient.ts

```diff
@@ -53,7 +53,7 @@ export async function sendRequest(request: WebRequest, options?: WebRequestOptio
     let timeToWait: number = retryIntervalInSeconds;
     while (true) {
         try {
-            if (request.body && typeof(request.body) !== 'string' && !request.body["readable"]) {
+            if (request.body && typeof(request.body) !== 'string') {
```

Comment:
If the stream is readable then we are forcefully picking creating a stream based on the contents of the request path right ? that could be empty ?

Reply:
Stream can be readable but the pointer might not point to the start of the stream, in this case the zip file which will start reading from where it left off in the last request.